conjur/config.py

```diff
@@ -100,19 +99,26 @@ def set(self, key, value):
 
     pubkeys_url = _service_url('pubkeys', doc='URL for the pubkeys service')
 
-    verify_ssl = _setting('verify_ssl', True,
-                          "Allows SSL verification to be disabled (development \
-                          only!)")
-    
```

Comment:
We don't allow `verify_ssl` to be turned off now, because cert handling is so smooth with present day conjur (i.e. just run conjur init).   Removing this setting also reduces some confusion in the tests about what the arguments to `requests.request` should look like, and lets us unify verification stuff in the `verify()` method below.